internal.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script is used by crc developer or internal CI to build the KAO/KCMO patch images to
+# brew and then mirror built container images to quay.io/crcont. Pull secret should contain
+# the auth for `quay.io/crcont` before providing to this script.
```

Comment:
maybe we can do a check for this with `jq` if the auth exists in the supplied pull-secret

Reply:
@anjannath yes we can do that and let dev know before hand (another improvement).